x-pack/plugins/siem/server/graphql/kpi_hosts/kpi_hosts.mock.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Logger } from '../../utils/logger';
+import { SiemContext } from '../index';
+import { KpiHostsData } from '../types';
+
+export const mockKpiHostsData: { KpiHosts: KpiHostsData } = {
+  KpiHosts: {
+    hosts: 0,
+    installedPackages: 0,
+    processCount: 0,
+    authenticationSuccess: 0,
+    authenticationFailure: 0,
+    fimEvents: 0,
+    auditdEvents: 0,
+    winlogbeatEvents: 0,
+    filebeatEvents: 0,
+    sockets: 0,
+    uniqueSourceIps: 0,
+    uniqueDestinationIps: 0,
+  },
+};
+
+export const getKpiHostsQueryMock = (logger: Logger) => ({
```

Comment:
Looks like this doesn't have any usages, is this intended as a helper for a future `schema` test?

Reply:
I think I will have to make a unit test for schema in this case 🤣